Web_Development/Javascript/Topics/5.Syntax/2.Operators/ComparisonOperator.md

```diff
@@ -0,0 +1,64 @@
+# Comparison Operators
+[Back to Operators in Javascript](README.md#operators-in-javascript)
+
+Comparison operators are used to compare values and returns boolean value (true or false).
+
+#### 1. Equal Operator (==)
+It will return true if both values are equal otherwise false.
+
+```js
+console.log(5==5); // true
+console.log(4==3); // false
+```
```

Comment:
Explain by an example that with == operator, JavaScript does not check for data types but only checks for values.
Example `5 == '5'` results in **true**.